tests/ScraperTest.php

```diff
@@ -259,4 +260,80 @@ public function testSingleQuotedMetaProperty()
         $this->assertEquals('13.11.2016 13:21:42', $meta->getModifiedTime()->format('d.m.Y H:i:s'));
         $this->assertEquals('Keyword1,Keyword2', $meta->getKeywords());
     }
+
+    public function testOgDomParser()
+    {
+        $data = <<<EOT
+            <title>Page title</title>
+            <meta name="description" content="Default page description"/>
+            <meta name="KEYWORDS" content="Keyword1,Keyword2">
+            <META NAME="author" CONTENT="Jozko Pucik"  >
+            <meta property="og:type" content="article" />
+            <meta property="og:description" content="Silny popis" />
+            <meta property="og:title" content="Og title nadpis"/>
+            <meta property="og:url" content="https://web.sk/stranka.html"/>
+            <meta property="og:site_name" content="Mega site name" />
+            <meta property="article:section" content="Ekonomika" />
+            <meta property="og:image" content="https://obrazok.jpg">
+            <meta property="article:published_time" content="2015-10-12T12:40:27+00:00" />
+            <meta property="article:modified_time" content="2016-11-13T13:21:42+00:00" />
+EOT;
+
+        $scraper = new Scraper();
+        $meta    = $scraper->parse($data, [new OgDomParser()]);
```

Comment:
can you remove these unnecessary spaces here and in other places as well?

Reply:
It is called "Align consecutive assignments" in code style section of IDE settings and I use them cause they improve readability in my oppinioin. But no problem - I'll remove it.

Reply:
thank you. it is more less based on personal preferences. but not sure if this is followed by PSR2 or PSR12

Reply:
> thank you. it is more less based on personal preferences. but not sure if this is followed by PSR2 or PSR12

there is nothing about alignment in PSR - so it's personal thing, as you said.